apps/docs/src/common/pages/patterns/access-a-service.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const applyAuthenticatePersonal = require('../../../../assets/images/p-authenticate-personal.svg').default;
+const applyAuthenticateReference = require('../../../../assets/images/p-authenticate-reference.svg').default;
+const applyAuthenticateTwoFa = require('../../../../assets/images/p-authenticate-2fa.svg').default;
+
+const Page: FC<PageProps> = ({ location }) => (
+<div className="govuk-grid-row">
+  <Helmet>
+    <title>Patterns - Home Office Design System</title>
```

Comment:
```
Access a service - Home Office Design System
```
maybe?
(Other pages will need similar changes.)
Basically this is what appears in the user's browser tab etc.

Reply:
Updated across all the pages